src/routes/about-us/+page.svelte

```diff
@@ -338,12 +338,12 @@
 				<div>
 					<div class="flex grid-cols-3 flex-wrap justify-center gap-5 lg:grid">
 						{#if communities.length}
-							{#each communities as community}
+							{#each communities as community (community.id)}
 								<AboutCommunity {community} />
 							{/each}
 						{:else}
 							<!-- eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars -->
```

Comment:
Can this be removed as well? (Maybe have a look if you find more occurrences which can be removed as well)